src/tiles/text-snippet-block.ts

```diff
@@ -90,32 +82,36 @@ export class TextSnippetBlock extends LitElement {
   static get styles(): CSSResultGroup {
     return css`
       .container {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        border-left: 5px solid #194880;
```

Comment:
Can we add a 3px border-radius too? (I think this was accidentally omitted from the spec document, but it's present in the images).

Reply:
that's weird because in current offshoot prod, adding it back or removing it doesn't do anything 🤔

Reply:
ohhhh... i notice it now, it's in the border-left corners

Reply:
updated now